src/Treasury.ts

```diff
@@ -35,4 +38,46 @@ export function handleSubscriptionPaid(event: FeesClaimed): void {
   movement.amount = event.params.jurorShare
   movement.createdAt = event.block.timestamp
   movement.save()
+  let subscriptions = Subscriptions.bind(event.address)
+  // TODO: it won't work if there's a token change after the event!!
```

Comment:
This sounds like it could be problem... is this case ever possible?

Reply:
Not sure about what you meant here, could you explain please?

Reply:
> is this case ever possible?
Definitely:
https://github.com/aragon/aragon-court/blob/master/contracts/subscriptions/CourtSubscriptions.sol#L254

Reply:
> Not sure about what you meant here, could you explain please?

Not sure if "you" here refers to Brett or me, but just in case, I didn't see an equivalent to [this function](https://github.com/aragon/aragon-court/blob/master/contracts/subscriptions/CourtSubscriptions.sol#L318) but for past configurations.

Reply:
I see 🤦‍♂ ... it sucks but we could use some of [these](https://github.com/aragon/aragon-court/blob/master/contracts/subscriptions/CourtSubscriptions.sol#L396) [functions](https://github.com/aragon/aragon-court/blob/master/contracts/subscriptions/CourtSubscriptions.sol#L419) to fetch that information maybe

Reply:
Oh, yes good point, I think the second one, `getJurorShare` would do the trick. Thanks!